File: solutions/go/etl/1/etl.go
package etl

import "strings"

func Transform(in map[int][]string) map[string]int {
	var out map[string]int = make(map[string]int)
    for k, v := range in {
        for _, i := range v {
            out[strings.ToLower(i)] = k
        }
    }
    return out
}

File: solutions/ruby/etl/1/etl.rb
=begin
Write your code for the 'ETL' exercise in this file. Make the tests in
`etl_test.rb` pass.

To get started with TDD, see the `README.md` file in your
`ruby/etl` directory.
=end

class ETL
  def self.transform(old)
    h=Hash.new
    old.each {|key, val| val.each {|v| h[v.downcase]=key } }
    h
  end
end